### ISSUE TITLE
Graphs and diagrams

### ISSUE THREAD

**Issue by chris-gds** [alphagov/frontend#5146]
## What 

Images and descriptive alt copy on [civil-service-diversity-and-inclusion-dashboard](https://www.gov.uk/government/publications/civil-service-diversity-inclusion-dashboard/civil-service-diversity-and-inclusion-dashboard) page

## Impact 

Fail of [WCAG 1.1.1](https://alphagov.github.io/wcag-primer/#guideline-1-1-provide-text-alternatives), [1.4.10](https://alphagov.github.io/wcag-primer/all.html#1-4-4-resize-text-aa)

### Consider:

- SVGs
- [Guidance](https://www.gov.uk/guidance/content-design/images)
- [Existing charts](https://components.publishing.service.gov.uk/component-guide/govspeak)
